src/components/primary-toolbar/primary-toolbar.test.js

```diff
@@ -37,6 +38,7 @@ describe('PrimaryToolbar', () => {
     ['.pipeline-menu-button--labels', 'onToggleTextLabels'],
     ['.pipeline-menu-button--export', 'onToggleExportModal'],
     ['.pipeline-menu-button--layers', 'onToggleLayers'],
+    ['.pipeline-menu-button--settings', 'onToggleSettingsModal'],
```

Comment:
While this function call is added, there has been no relevant test to use and test this in `mapDispatchToProps` - please add that test

Reply:
I saw that a test has been added, hence resolving this.